jdi-light-mobile-examples/src/main/java/selenium/entities/TestData.java

```diff
@@ -0,0 +1,405 @@
+package selenium.entities;
+
+public class TestData {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestData' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)